### PR TITLE
Add information to moving monitoring components instructions

### PR DESCRIPTION
**Pull request by rh-max**

### CODE REVIEW

**Comment by rh-max** [comment]
@ahardin-rh Could you PTAL and merge & cherry-pick to `enterprise-4.2` and `enterprise-4.3`? Thank you.

**Comment by rh-max** [comment]
@bmcelvee Could you please review this and merge & cherry-pick to `enterprise-` 4.2, 4.3, and 4.4? Thanks.

File: modules/infrastructure-moving-monitoring.adoc [hunk 1]
```diff
@@ -62,4 +65,13 @@ $ oc create -f cluster-monitoring-configmap.yaml
...
+----
+
+. If a component has not moved to the `infra` node, delete the pod with this
+component:
++
+----
+$ oc delete pod -n openshift-monitoring <pod>
+----
++
+The component from the deleted pod will now be re-created on the `infra` node.
```
Comment (bmcelvee) [inline modules/infrastructure-moving-monitoring.adoc:77]: For consistency with recommendations I've had in the past: 

s/will now be/is

**Comment by bmcelvee** [comment]
One tiny change request, then I'll be happy to merge and cherry pick. :)

**Comment by rh-max** [comment]
@bmcelvee Done. Thank you!

**Pull request merged**